scintillometry/channelize/base.py

```diff
@@ -0,0 +1,93 @@
+# Licensed under the GPLv3 - see LICENSE
+
+import operator
+from astropy.utils import lazyproperty
+
+from ..base.base import ModuleBase
+from ..fourier import get_fft_maker
+
+
+__all__ = ['ChannelizeModule']
+
+
+class Channelizer(object):
```

Comment:
I still am not sure there is a point in having this separate class.... Maybe keep it simple for now?

Reply:
The original thought was `Channelizer` could also be used in `DedisperseModule`, but largely agreed this is an unnecessary abstraction for now (especially since the actual core is the `fft`, which is already abstracted).  Changed.